libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -23,7 +23,7 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
     height: $gd-header-height;
     padding: 0 0 0 20px;
     box-shadow: 0 1px 3px rgba(0, 0, 0, 0.35);
-    color: $gd-color-text-light;
+    color: #fff;
```

Comment:
this is intentional?

Reply:
yes, the header should not be affected by complementary palette